<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1253</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1261</width>
      <height>791</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="ShowInterpolation">
     <attribute name="title">
      <string>Interpolation</string>
     </attribute>
     <widget class="QCustomPlot" name="InterpolationGraphView" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1221</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>700</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Points for plotting</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InterLoadDataBut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>700</y>
        <width>151</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Load data</string>
      </property>
     </widget>
     <widget class="QPushButton" name="InterClearGraphBut">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>700</y>
        <width>151</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear graph</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>700</y>
        <width>110</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choice datatime</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InterPromotionValue">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>720</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>700</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Promotion predict</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="InterSourceFile">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>720</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>700</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Source file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DateForPromotionInterpolation">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>720</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>YYYY-MM-DD</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="cursorPosition">
       <number>10</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QSpinBox" name="InterPointsForPlotting">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>720</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
     <widget class="QPushButton" name="InterPlotGrafBut">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>700</y>
        <width>151</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot Graf</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ShowApproximation">
     <attribute name="title">
      <string>Approximation</string>
     </attribute>
     <widget class="QCustomPlot" name="ApproximationGraphView" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1221</width>
        <height>651</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>700</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Promotion predict</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>700</y>
        <width>110</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Choice datatime</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ApproxPromotionValue">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>720</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ApproxClearGraphBut">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>700</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear graph</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ApproxInterLoadDataBut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>700</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Load data</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>700</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Points for plotting</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ApproxSourceFile">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>720</y>
        <width>381</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>860</x>
        <y>700</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Source file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DateForPromotionApproximation">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>720</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>YYYY-MM-DD</string>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="ApproxPlotGrafBut">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>700</y>
        <width>101</width>
        <height>50</height>
       </rect>
      </property>
      <property name="text">
       <string>Plot Graf</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ApproxPointsForPlotting">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>720</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>500000</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="ApproxCoefficient">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>720</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>700</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Coefficient</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="ApproxDayPred">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>720</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="maximum">
       <number>0</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>700</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Day to pred</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
